osu.Game/Screens/OnlinePlay/Lounge/Components/DrawableRoom.cs

```diff
@@ -101,6 +103,25 @@ public bool MatchingFilter
             }
         }
 
+        private int numberOfAvatars = 3;
+
+        public int NumberOfAvatars
```

Comment:
Is this actually going to be something dynamic in the design? Feels unnecessary if not.

Reply:
In the designs it's variable - the in-match-subscreen has 3 avatars while the lounge has 7. I suppose this is because there's more space in the lounge whereas the in-match version has additional buttons.
The host text is also a bit misplaced which would otherwise further reduce the available width, but as I mentioned in the OP I've placed it where it is because we don't have a description and the details on the left would look weird.